wagtail/users/tests/test_admin_views.py

```diff
@@ -1806,3 +1806,257 @@ def test_get_group_viewset_cls_custom_form_does_not_exist(self):
                 'Module "wagtail.users.tests" does not define a "CustomClassDoesNotExist" attribute/class',
                 str(exc_info.exception),
             )
+
+
+class TestAuthorizationIndexView(TestCase, WagtailTestUtils):
+    def setUp(self):
+        self._user = self.create_user(username="auth_user", password="password")
+        self._user.user_permissions.add(Permission.objects.get(codename="access_admin"))
+        self.login(username="auth_user", password="password")
+
+    def get(self, params={}):
+        return self.client.get(reverse("wagtailusers_users:index"))
+
+    def gain_permissions(self):
+        self._user.user_permissions.add(
+            *Permission.objects.filter(
+                content_type__app_label=AUTH_USER_APP_LABEL,
+                codename__in=(
+                    "add_{}".format(AUTH_USER_MODEL_NAME.lower()),
+                    "change_{}".format(AUTH_USER_MODEL_NAME.lower()),
+                    "delete_{}".format(AUTH_USER_MODEL_NAME.lower()),
+                ),
+            )
+        )
+
+    def test_simple(self):
+        response = self.get()
+        self.assertEqual(response.status_code, 302)
+        self.assertEqual(
+            response.context["message"],
+            "Sorry, you do not have permission to access this area.",
+        )
+
+    def test_authorized(self):
+        self.gain_permissions()
+        response = self.get()
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, "wagtailusers/users/index.html")
+
+
+class TestAuthorizationCreateView(TestCase, WagtailTestUtils):
+    def setUp(self):
+        self._user = self.create_user(username="auth_user", password="password")
+        self._user.user_permissions.add(Permission.objects.get(codename="access_admin"))
+        self.login(username="auth_user", password="password")
+
+    def get(self, params={}):
+        return self.client.get(reverse("wagtailusers_users:add"), params)
+
+    def post(self, post_data={}):
+        return self.client.post(reverse("wagtailusers_users:add"), post_data)
+
+    def gain_permissions(self):
+        self._user.user_permissions.add(
+            Permission.objects.get(
+                content_type__app_label=AUTH_USER_APP_LABEL,
+                codename="add_{}".format(AUTH_USER_MODEL_NAME.lower()),
+            )
+        )
+
+    def test_simple(self):
+        response = self.get()
+        self.assertEqual(response.status_code, 302)
+        self.assertEqual(
+            response.context["message"],
+            "Sorry, you do not have permission to access this area.",
+        )
+
+    def test_authorized(self):
```

Comment:
```suggestion
    def test_authorised(self):
```

Similar here and the others, using UK spelling.